.changes/1.25.0.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "category": "``lookoutmetrics``",
+    "description": "Adding backtest mode to detectors using the Cloudwatch data source.",
+    "type": "api-change"
+  },
+  {
+    "category": "``transcribe``",
+    "description": "Amazon Transcribe now supports automatic language identification for multi-lingual audio in batch mode.",
+    "type": "api-change"
+  },
+  {
+    "category": "``iotsitewise``",
+    "description": "This release adds the following new optional field to the IoT SiteWise asset resource: assetDescription.",
+    "type": "api-change"
+  },
+  {
+    "category": "``sagemaker``",
+    "description": "Amazon SageMaker Notebook Instances now support Jupyter Lab 3.",
+    "type": "api-change"
+  },
+  {
+    "category": "Python",
+    "description": "Dropped support for Python 3.6",
+    "type": "feature"
+  },
+  {
+    "category": "``drs``",
+    "description": "Changed existing APIs and added new APIs to accommodate using multiple AWS accounts with AWS Elastic Disaster Recovery.",
+    "type": "api-change"
+  },
+  {
+    "category": "``cognito-idp``",
+    "description": "Amazon Cognito now supports IP Address propagation for all unauthenticated APIs (e.g. SignUp, ForgotPassword).",
+    "type": "api-change"
+  },
+  {
+    "category": "Python",
+    "description": "Dropped support for Python 3.6",
```

Comment:
Looks like this entry got added twice.

Reply:
Hmm, this is a quirk of the CLI not denoting whether a change is for itself or from Botocore.